.github/workflows/swift-toolchain.yml

```diff
@@ -3193,6 +3193,34 @@ jobs:
             Install-Package -Name WixToolset.Sdk -RequiredVersion 4.0.1 -Force
           }
 
+      - uses: actions/checkout@v4
+        if: matrix.arch == 'amd64'
+        with:
+          repository: microsoft/mimalloc
+          ref: refs/tags/v2.1.7
+          path: ${{ github.workspace }}/SourceCache/mimalloc
+          show-progress: false
+
+      - name: Build and apply mimalloc
+        if: matrix.arch == 'amd64'
+        run: |
+          # Reference: https://github.com/microsoft/mimalloc/tree/dev/bin#minject
+          msbuild ${{ github.workspace }}\SourceCache\mimalloc\ide\vs2022\mimalloc.sln -p:Configuration=Release
+          $ToolchainBin = "${{ github.workspace }}\BuildRoot\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin"
+          Copy-Item -Path "${{ github.workspace }}\SourceCache\mimalloc\out\msvc-x64\Release\mimalloc-override.dll" `
+            -Destination "$ToolchainBin"
+          Copy-Item -Path "${{ github.workspace }}\SourceCache\mimalloc\out\msvc-x64\Release\mimalloc-redirect.dll" `
+            -Destination "$ToolchainBin"
+          $MimallocExecutables = @("swift.exe","swiftc.exe","swift-driver.exe","swift-frontend.exe")
+          $MimallocExecutables += @("clang.exe","clang++.exe","clang-cl.exe")
+          $MimallocExecutables += @("lld.exe","lld-link.exe","ld.lld.exe","ld64.lld.exe")
```

Comment:
Why not create a single list with a single element per line?

Reply:
Done